src/winstanley/WdlBraceMatcher.scala

```diff
@@ -0,0 +1,20 @@
+package winstanley
+
+import com.intellij.lang.{BracePair, PairedBraceMatcher}
+import com.intellij.psi.PsiFile
+import com.intellij.psi.tree.IElementType
+import winstanley.psi.WdlTypes
+
+class WdlBraceMatcher extends PairedBraceMatcher {
+  override def isPairedBracesAllowedBeforeType(iElementType: IElementType, iElementType1: IElementType): Boolean = true
+
+  override def getPairs: Array[BracePair] = Array(
+    new BracePair(WdlTypes.COMMAND_VAR_OPENER, WdlTypes.RBRACE, false),
```

Comment:
I assume the asymmetry is because this is `${` and `}`?

Reply:
Right!

I suspect we could make the `RBRACE` here a `COMMAND_VAR_CLOSER` token with some smarter lexing rules.. OTOH this works so I'm inclined to leave it as-is for now! 🤷‍♂️